package/autoyast2.changes

```diff
@@ -1,4 +1,9 @@
 -------------------------------------------------------------------
+Fri Jan 29 09:04:47 UTC 2016 - igonzalezsosa@suse.com
+
+- "haspcmcia" method is removed from the AutoInstallRules API.
```

Comment:
It's a dead code removal, but we might have issues with BS guys when accepting to SP2. I suggest to create a `cleanup bug` and describe it there. It's just a few clicks, but could help us a lot :) Thx